src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+  const DAILY_COST = 40;
+  let totalAmount = days * DAILY_COST;
+
+  if (days >= LONG_TERM) {
+    totalAmount -= LONG_TERM_DISCOUNT;
+
+    return totalAmount;
+  }
```

Comment:
You don't have to change the value of `totalAmount`; instead, return the result of the operation. This way, `totalAmount` can be a constant.
```suggestion
  const totalAmount = days * DAILY_COST;

  if (days >= LONG_TERM) {
     return totalAmount - LONG_TERM_DISCOUNT;
  }
```